setup.py

```diff
@@ -15,32 +13,43 @@
     "xarray",
 ]
 
-test_requirements = ["pytest", "pytest-subtests", "serialbox", "coverage"]
-ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@develop"]
-develop_requirements = test_requirements + ndsl_requirements + ["pre-commit"]
+test_requirements = [
+    "coverage",
+    "pytest",
+    "pytest-subtests",
+    "serialbox",
+]
+
+ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@2024.09.00"]
+
+develop_requirements = [
+    *ndsl_requirements,
+    *test_requirements,
+    "pre-commit",
+]
 
 extras_requires = {
-    "test": test_requirements,
-    "ndsl": ndsl_requirements,
     "develop": develop_requirements,
+    "ndsl": ndsl_requirements,
+    "test": test_requirements,
 }
 
 setup(
-    author="The Allen Institute for Artificial Intelligence",
+    author="NOAA - Geophysical Fluid Dynamics Laboratory",
     author_email="oliver.elbert@noaa.gov",
-    python_requires=">=3.11",
+    python_requires=">=3.11,<3.12",
     classifiers=[
         "Development Status :: 2 - Pre-Alpha",
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: BSD License",
+        "License :: OSI Approved :: GPLv3 License",
         "Natural Language :: English",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.11",
     ],
-    description="pyFV3 is a NDSL-based FV3 dynamical core for atmospheric models",
+    description="PyFV3 is a NDSL-based FV3 dynamical core for atmospheric models.",
     install_requires=requirements,
     extras_require=extras_requires,
-    license="BSD license",
+    license="GPLv3 license",
     long_description=readme,
     include_package_data=True,
```

Comment:
Actually, I have a question on that line now that I see it. Does anyone know what the intention was here? While the default in the newer `project.toml` files is `True`, the default in `setup.py` is (and remains for compatibility reasons) `False`.

As I understand [the docs](https://setuptools.pypa.io/en/latest/userguide/quickstart.html#including-data-files), for this to have an effect, we'd need to provide a `MANIFEST.in` file or configure the git plugin to automatically add data files to the package...

Reply:
I think a while back (at AI2) the idea was to ship some translate data with the package. This was swiftly abandoned.

Reply:
In that case, let me just delete that line to avoid any future confusion